core/src/components/picker-column-internal/picker-column-internal.tsx

```diff
@@ -254,12 +254,13 @@ export class PickerColumnInternal implements ComponentInterface {
         const centerX = bbox.x + bbox.width / 2;
         const centerY = bbox.y + bbox.height / 2;
 
-        const activeElement = el.shadowRoot!.elementFromPoint(centerX, centerY) as HTMLButtonElement;
+        const activeElement = el.shadowRoot!.elementFromPoint(centerX, centerY) as HTMLButtonElement | null;
+
         if (activeEl !== null) {
           activeEl.classList.remove(PICKER_COL_ACTIVE);
         }
 
-        if (activeElement.disabled) {
+        if (!activeElement || activeElement.disabled) {
```

Comment:
Can we do an explicit null check like we do in line 258?
```suggestion
        if (activeElement === null || activeElement.disabled) {
```